scripts/setup.py

```diff
@@ -0,0 +1,13 @@
+from setuptools import setup
+
+setup(
+    name='securedrop',
+    version='1.0',
+    packages=['securedrop', 'securedrop.tests'],
+    url='https://github.com/aokellermann/securedrop.git',
+    license='MIT License',
+    author='John Nay, Antony Kellermann, Robert Liani',
```

Comment:
This should be the name of the person who "owns" the project, which is me since it is hosted on my github account.

Reply:
Changed.